persia/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.1.dev13"
\ No newline at end of file
+__version__ = "0.1.dev13"
```

Comment:
don't push change to this file

Reply:
Solved in https://github.com/PersiaML/PersiaML/commit/6f8ad9c8f62a832b57813876511a96fdc455a3d8